astroquery/solarsystem/pds/tests/test_ringnode.py

```diff
@@ -0,0 +1,303 @@
+import pytest
+import os
+import numpy as np
+
+from astropy.tests.helper import assert_quantity_allclose
+import astropy.units as u
+
+from astroquery.utils.mocks import MockResponse
+from ... import pds
+
+# files in data/ for different planets
+DATA_FILES = {'Uranus': 'uranus_ephemeris.html',
+              'Pluto': 'pluto_ephemeris.html',
+              'Neptune': 'neptune_ephemeris.html',
+              'Saturn': 'saturn_ephemeris.html',
+              }
+
+
+def data_path(filename):
+    data_dir = os.path.join(os.path.dirname(__file__), "data")
+    return os.path.join(data_dir, filename)
+
+
+# monkeypatch replacement request function
+def nonremote_request(self, request_type, url, **kwargs):
+
+    planet_name = kwargs['params']['center_body']
+    with open(data_path(DATA_FILES[planet_name.capitalize()]), "rb") as f:
+        response = MockResponse(content=f.read(), url=url)
+
+    return response
+
+
+# use a pytest fixture to create a dummy 'requests.get' function,
+# that mocks(monkeypatches) the actual 'requests.get' function:
+@pytest.fixture
+def patch_request(request):
+    mp = request.getfixturevalue("monkeypatch")
+
+    mp.setattr(pds.core.RingNodeClass, "_request", nonremote_request)
+    return mp
+
+
+# --------------------------------- actual test functions
+
+def test_ephemeris_query_Uranus(patch_request):
+
+    systemtable, bodytable, ringtable = pds.RingNode.ephemeris(
+        planet="Uranus",
+        epoch="2022-05-03 00:00",
+        location=(10.0 * u.deg, -120.355 * u.deg, 1000 * u.m),
+    )
+    # check system table
+    assert_quantity_allclose(
+        [
+            -56.12233,
+            -56.13586,
+            -56.13586,
+            -56.01577,
+            0.10924,
+            354.11072,
+            354.12204,
+            2947896667.0,
+            3098568884.0,
+            10335.713263,
+        ],
+        [
+            systemtable["sub_sun_lat"].to(u.deg).value,
+            systemtable["sub_sun_lat_min"].to(u.deg).value,
+            systemtable["sub_sun_lat_max"].to(u.deg).value,
+            systemtable["opening_angle"].to(u.deg).value,
+            systemtable["phase_angle"].to(u.deg).value,
+            systemtable["sub_sun_lon"].to(u.deg).value,
+            systemtable["sub_obs_lon"].to(u.deg).value,
+            systemtable["d_sun"].to(u.km).value,
+            systemtable["d_obs"].to(u.km).value,
+            systemtable["light_time"].to(u.second).value,
+        ],
```

Comment:
The point of using `assert_quantity_allclose()` is to compare both the values and the units. If you are comparing the numerical values only then you would be better off using `np.allclose()`.